lib/actions/user.js

```diff
@@ -31,10 +31,12 @@ function createNewUser (auth0User) {
     auth0UserId: auth0User.sub,
     email: auth0User.email,
     hasConsentedToTerms: false, // User must agree to terms.
+    language: 'fr',
```

Comment:
This should probably not be hardcoded. It seems like something like this would be derived from the current locale setting that the user selected in the app.

Reply:
Removed in https://github.com/opentripplanner/otp-react-redux/pull/375/commits/2f66ec4f23cc34feb530cac5621277dc7f82e6cb.